#11 Added support for VIDEO-RANGE attribute for Variants

simonerni opened the pull request:
As discussed in #11

carlanton commented:
Hi!

This looks great. I'd like if you could:

 1. Add the tag to a variant under `test/resources/master/master.m3u8`. There are some tests that will make sure that all tags can be rendered correctly.

2. Add the tag to the README under Supported tags.

Thanks,
Anton

simonerni commented:
> 1. Add the new attribute to a variant under test/resources/master/master.m3u8. There are some tests that will make sure that all tags can be rendered correctly.

Sure thing, I added it to the master.m3u8 file. I also added support for I Frame playlist variants, as they support the video range attribute as well.
> 2. Add the new attribute to the README under Supported tags.

Im not sure if it's appropriate for the supported tags section, as it's an attribute, not a tag.

Kind Regards,
Simon

carlanton commented:
Perfect :) Thanks a lot!
